backend/app/api/endpoints/adapter/shells.py

```
@@ -0,0 +1,505 @@
+    shell.json = shell_crd.model_dump(mode="json")
+    db.add(shell)
+    db.commit()
+    db.refresh(shell)
+
+    return _user_shell_to_unified(shell).model_dump()
+
+
+@router.delete("/{shell_name}")
+def delete_shell(
+    shell_name: str,
+    db: Session = Depends(get_db),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)